features/payment/managing_payment_methods/editing_payment_method.feature

```diff
@@ -0,0 +1,47 @@
+@managing_payment_methods
+Feature: Editing payment methods
+    In order to change which payment methods are available in my store
+    As an Administrator
+    I want to be able to edit payment method
+
+    Background:
+        Given the store has a payment method "Offline" with a code "OFF"
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Renaming the payment method
+        Given I want to modify a payment method "Offline"
+        When I rename it to "Cash on delivery" in "English (United States)"
+        And I save my changes
+        Then I should be notified that it has been successfully edited
+        And this payment method name should be "Cash on delivery"
+
+    @ui
+    Scenario: Modifying payment method gateway
+        Given I want to modify a payment method "Offline"
+        When I choose "Paypal Express Checkout" gateway
+        And I save my changes
+        Then I should be notified that it has been successfully edited
+        And this payment method gateway should be "paypal_express_checkout"
+
+    @ui
+    Scenario: Disabling payment method
+        Given I want to modify a payment method "Offline"
+        When I disable it
+        And I save my changes
+        Then I should be notified that it has been successfully edited
+        And this payment method should be disabled
+
+    @ui
+    Scenario: Enabling payment method
+        Given the store has a payment method "Offline" disabled
+        And I want to modify a payment method "Offline"
```

Comment:
And I want to modify ~~a~~ **the "Offline"** payment method ~~"Offline"~~

Correct this everywhere else also